build/packaging/linux/bin/kscnd

```diff
@@ -304,9 +304,12 @@ start_node() {
         fi
     fi
 
-    if [[ ! -z $VTRECOVERY ]] && [[ $VTRECOVERY -eq 1 ]]; then
+    if [[ ! -z $SC_VTRECOVERY ]] && [[ $SC_VTRECOVERY -eq 1 ]]; then
         OPTIONS="$OPTIONS --vtrecovery"
     fi
+    if [[ ! -z $SC_VTRECOVERY_INTERVAL ]]; then
+        OPTIONS="$OPTIONS --vtrecoveryinterval $SC_VTRECOVERY_INTERVAL"
+    fi
```

Comment:
```suggestion
    if [[ ! -z $SC_VTRECOVERY ]] && [[ $SC_VTRECOVERY -eq 1 ]]; then
        OPTIONS="$OPTIONS --vtrecovery"
        if [[ ! -z $SC_VTRECOVERY_INTERVAL ]]; then
            OPTIONS="$OPTIONS --vtrecoveryinterval $SC_VTRECOVERY_INTERVAL"
        fi
    fi
    
```

Reply:
It seems that SC_VTRECOVERY_INTERVAL has some value only if SC_VTRECOVERY=1. Then why don't you make this logic to nested if statement?

Reply:
Thank you. PTAL again. I found other typos too and fixed them, thanks to your comment.